<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GoldenRatioWidget</class>
 <widget class="QWidget" name="GoldenRatioWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>125</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>8</number>
   </property>
   <property name="topMargin">
    <number>8</number>
   </property>
   <property name="rightMargin">
    <number>8</number>
   </property>
   <property name="bottomMargin">
    <number>8</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="0" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,1,0">
     <property name="spacing">
      <number>4</number>
     </property>
     <item>
      <widget class="QLineEdit" name="sourceValue">
       <property name="placeholderText">
        <string>source value</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="type">
       <item>
        <property name="text">
         <string>Long</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Short</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Whole</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="ratioKoef">
       <property name="text">
        <string>x 1.61803399</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="2">
    <widget class="QLineEdit" name="wholeValue">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(239, 239, 239);</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="longLabel">
     <property name="text">
      <string>Long</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="wholeLabel">
     <property name="text">
      <string>Whole</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLineEdit" name="longValue">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(239, 239, 239);</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="shortLabel">
     <property name="text">
      <string>Short</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="shortValue">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(239, 239, 239);</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>sourceValue</tabstop>
  <tabstop>type</tabstop>
  <tabstop>longValue</tabstop>
  <tabstop>shortValue</tabstop>
  <tabstop>wholeValue</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
